### PR TITLE
Add the cloakroom icon

### PR DESCRIPTION
**Pull request by chucklehound**
+ [/] I've updated `changelog.md` for any changes that need to be highlighted in the changelog.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ _Nothing yet!_
   - Added new outputs for the iOS and Android platforms in `bpk-tokens/tokens/ios/base.ios.json`, `bpk-tokens/tokens/ios/base.react.native.es6.js`, `bpk-tokens/tokens/android/base.android.xml` and
   `bpk-tokens/tokens/android/base.react.native.es6.js`. These interfaces are not to be considered stable yet however and they may still change.
 
+## 2017-08-16 - New icon for cloakroom
```
Comment (georgegillams) [inline changelog.md]: This should not be dated as we have not released the new icon.

Your changelog entry should simply be this under the `## Unreleased` title

```

**Added:**
- bpk-component-icon:
- bpk-mixins:
- bpk-svgs:
  - added 1 new icon (cloakroom)
```

**Comment by chucklehound** [comment]
Doh!  That's what happens when I wait overnight to finish committing :)

I'll get the rebase and changelog sorted, but may need to get some help on squashing all these commits (unless you're fine with them not being squashed)

**Comment by georgegillams** [comment]
I can deal with this if you'd prefer?

**Comment by chucklehound** [comment]
@georgegillams Only if it's easier.  I need to learn how to do this one day :)

**Comment by georgegillams** [comment]
I've got the changes ready to go :)

**Comment by chucklehound** [comment]
Haha!  Well remind me to buy you a pint as a thank you sometime :)

**Comment by georgegillams** [comment]
Ah - I don't have permission to push to your repo!

**Comment by chucklehound** [comment]
Just invited you.

**Pull request merged**